packages/web-twig/src/Resources/components/Item/Item.twig

```diff
@@ -57,9 +57,10 @@
     {%- endif -%}
   </span>
   <HelperText
-    elementType="span"
     className="{{ _helperTextClassName }}"
+    elementType="span"
     helperText="{{ _helperText }}"
+    id="{{ 'item-helper-text' }}"
```

Comment:
Temporary state. We shouldn't use same ID on the page more than once…

I'm not sure how to handle passing `id` if the component `Item` doesn't accept one. Use label in the kebab-case syntax, or extend API?

Reply:
I have tried to do this the same way as in all other components, but since the `is` is not mandatory it is hard to say. Maybe the easiest way to fix this is just to set `id` as required for the `Item` component.

Reply:
I made the ValidationText and HelperText optional, so this is no longer needed. See #1470